client/src/components/ProductInformation/index.jsx

```diff
@@ -3,26 +3,32 @@ import PropTypes from 'prop-types';
 import styles from './styles.css';
 
 const ProductInformation = ({ productInfo, originalPrice, salePrice }) => {
-  const currentPrice = salePrice === null ? originalPrice
-    : (
-      <span className={styles.originalPrice}>
+  const currentPrice = salePrice === null ? (
+    <div className={styles.price}>
+      $
+      {originalPrice}
+    </div>
+  ) : (
+    <div>
+      <div className={styles.salePrice}>
+        $
+        {salePrice}
+      </div>
+      <div className={styles.originalPrice}>
         {originalPrice}
-      </span>
-    );
+      </div>
+    </div>
+  );
 
   return (
     <div className={`${styles.productInfo} ${styles.productInformation}`}>
-      <div className={styles.ratings}>Star Ratings</div>
+      <div className={styles.ratings}>
+        <span>Star Ratings </span>
+        <span>Read All Reviews</span>
+      </div>
       <div className={styles.category}>{productInfo.category}</div>
       <div className={styles.name}>{productInfo.name}</div>
-      <div className={styles.salePrice}>
-        $
-        {salePrice}
-      </div>
-      <div className={styles.price}>
-        $
-        {currentPrice}
-      </div>
+      {currentPrice && currentPrice}
```

Comment:
seems to me currentPrice will always exist?
```suggestion
      {currentPrice}
```

Reply:
That's right! Let me correct that :)